### PR TITLE
Add `alphanumeric` character set

### PR DESCRIPTION
**Pull request by Richienb**
Fixes: #23 

// @sindresorhus

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -35,7 +37,7 @@ interface TypeOption {
 	//=> 'CDEHKM'
```
Comment (sindresorhus) [inline index.d.ts:37]: You need to add example here too like readme

**Comment by sindresorhus** [comment]
Can you also mention your use-case in the docs about it? It's often useful as users don't always know what they would use something for.

**Comment by sindresorhus** [comment]
Bump :)

**Comment by Richienb** [comment]
@sindresorhus Ready for re-review!

**Comment by sindresorhus** [comment]
> https://github.com/sindresorhus/crypto-random-string/pull/25/files#r474970876

:arrow_up:

File: readme.md [hunk 2]
```diff
@@ -33,6 +33,9 @@ cryptoRandomString({length: 6, type: 'distinguishable'});
 cryptoRandomString({length: 10, type: 'ascii-printable'});
 //=> '`#Rt8$IK>B'
 
+cryptoRandomString({length: 10, type: 'alphanumeric'});
+//=> 'DMuKL8YtE7'
```
Comment (sindresorhus) [inline readme.md:37]: index.d.ts and the readme should be in sync (please try to ensure this is the case for every PR).

**Comment by sindresorhus** [comment]
Bump

**Comment by Richienb** [comment]
@sindresorhus Ready for review!

**Pull request merged**